src/EditorFeatures/Test2/FindReferences/FindReferencesTests.PropertySymbols.vb

```diff
@@ -945,6 +945,28 @@ namespace ConsoleApplication22
 }
         </Document>
     </Project>
+</Workspace>
+            Await TestAPIAndFeature(input, kind, host)
+        End Function
+
+        <WorkItem(44288, "https://github.com/dotnet/roslyn/issues/44288")>
+        <WpfTheory, CombinatorialData, Trait(Traits.Feature, Traits.Features.FindReferences)>
+        Public Async Function TestPropertyReferenceInGlobalSuppression(kind As TestKind, host As TestHost) As Task
+            Dim input =
+<Workspace>
+    <Project Language="C#" CommonReferences="true">
+        <Document>
+
+[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Category", "RuleId", Scope = "member", Target = "~P:N.C.[|P|]")]
+
+namespace N
+{
+    class C
+    {
+        public int {|Definition:$$P|} { get; set; }
+}
```

Comment:
test for parameterized property?
test for indexers?

Reply:
Added, thanks!